study-builder/studies/testboston/emails/next_kit2.tmpl.html

```diff
@@ -0,0 +1,224 @@
+                  <tr>
+                    <td>
+                      <!--[if mso]>
+                      <center>
+                        <table><tr><td width="600">
+                      <![endif]-->
+                      <table width="100%" cellpadding="0" cellspacing="0" border="0" style="width:100%; max-width:600px;" align="center">
+                        <tr>
+                          <td role="modules-container" style="padding:0px 0px 0px 0px; color:#464f61; text-align:left;" bgcolor="#ffffff" width="100%" align="left"><table class="module preheader preheader-hide" role="module" data-type="preheader" border="0" cellpadding="0" cellspacing="0" width="100%" style="display: none !important; mso-hide: all; visibility: hidden; opacity: 0; color: transparent; height: 0; width: 0;">
+                          </table><table class="wrapper" role="module" data-type="image" border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;" data-muid="4UqFsRLozLcypAAv4CeoFS">
+                            <tbody><tr>
+                              <td style="font-size:6px; line-height:10px; padding:30px 0px 10px 18px;" valign="top" align="left">
+                                <img class="max-width" border="0" style="display:block; color:#000000; text-decoration:none; font-family:Helvetica, arial, sans-serif; font-size:16px; max-width:50% !important; width:50%; height:auto !important;" src="{{baseWebUrl}}/assets/images/logo-testboston.png" alt="TestBoston" width="300" data-responsive="true" data-proportionally-constrained="false">
+                              </td>
+                            </tr>
+                            </tbody></table><table class="module" role="module" data-type="text" border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;" data-muid="13fe2859-7e54-40df-8fd6-536a6cf6edbb" data-mc-module-version="2019-10-22">
+                            <tbody>
+                            <tr>
+                              <td style="padding:20px 20px 20px 20px; line-height:24px; text-align:inherit; background-color:#ffffff;" height="100%" valign="top" bgcolor="#ffffff" role="module-content"><div><div style="font-family: inherit; text-align: inherit">${text["salutation"]}</div>
+                                <div style="font-family: inherit; text-align: inherit"><br></div>
+                                <div style="font-family: inherit; text-align: inherit; color: #464f61;">${text["kit_coming"]}</div>
+                                <br>
+                                <div style="font-family: inherit; text-align: inherit; color: #464f61;">${text["next_week"]}</div>
+                                <br>
+                                <div style="font-family: inherit; text-align: inherit; color: #464f61;">${text["update_address"]} <a href="{{baseWebUrl}}/mailing-address">${text["click_here"]}</a>.</div>
+                                <br>
```

Comment:
Created a new `*.tmpl.html` file so I can get a new paragraph in here. Otherwise it's a copy-paste of `next_kit.tmpl.html` file.